image-meta-tag/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+    name: image-meta-tag
+    version: "0.4"
+
+source:
+    git_url: https://github.com/SciTools-incubator/image-meta-tag
+    git_tag: 0.4.1
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python
```

Comment:
The tests seem to pass with Python 2.7 but not with 3.4, so a switch can be put in here to test only with Python less than 3:

- python < 3